### DISCUSSION TITLE
[SOLVED] How do you select and move multiple feeds into one category?

### DISCUSSION THREAD

**Discussion by formlessdao** [martinrotter/rssguard#524] [Q&A]
Am not able to do what i said in the title. And by multiple feeds I mean over 3000 feeds that I can not select one by one. Thanks!

**Comment by martinrotter** [comment]
Not currently possible.

**Comment by martinrotter** [comment]
https://github.com/martinrotter/rssguard/issues/143

**Reply by formlessdao** [reply]
Why was it not implemented since 2017?

**Reply by martinrotter** [reply]
> RSS Guard is an **open-source** cross-platform multi-protocol desktop feed reader

Why? No one provided pull request with such a feature. The ticket itself is basically dead, there just does not seem to be great interest in the feature from big % of users.

You are welcome to provide pull request with the feature or somehow hire or pay someone to contribute it. I do not feel the need ot do it by myself right now.

**Comment by akinokonomi** [comment]
Well, I think it actually _is_ possible if you use standard feed account (not Feedly, etc.).


**Note that you'll lose articles of all feeds you already have.**

0. Do a backup of RSS Guard config directory (just in case something goes wrong)
1. Right click your account, e.g. "Username (RSS/ATOM/JSON)"
2. Click "Export feeds" and export them wherever you want
3. Open generated `.opml` in text editor (the scheme is easy to understand), and move the feeds around as you like
4. Import your edited `.opml` to new standard account

It's hacky and roundabout, but it should work. 


**And also note** that you are likely to have huge performance issues while editing those files, because icons are also embedded in there, and sometimes one line can be 60-thousand-characters long.

So I strongly advice you to:
1. Use something like Sublime Text (perhaps Notepad++ will also be fine)
2. Disable syntax highlighting
3. Disable line wrapping
4. *Move* lines—**don't copy-paste**. (In Notepad++ and Sublime it is `Ctrl+Shift+(Up|Down)` to move selected lines.)


Though even with all this it is very unlikely that you will be able to edit those files manually, since you have ~3000 feeds.

**Reply by akinokonomi** [reply]
You can use some OPML editor, like [buzz](http://buzz.sourceforge.net/). 
But from here you are on your own, I know nothing about OPML.
